dask/array/tests/test_reductions.py

```diff
@@ -459,12 +459,6 @@ def test_topk_argtopk1(npfunc, daskfunc, split_every):
     with pytest.raises(ValueError):
         daskfunc(b, -k, axis=3, split_every=split_every)
 
-    # structured arrays
-    assert_eq(npfunc(c, axis=0)[-1:][::-1],
-              daskfunc(c, 1, split_every=split_every))
-    assert_eq(npfunc(c, axis=0)[:1],
-              daskfunc(c, -1, split_every=split_every))
```

Comment:
Why were these tests removed?  Were they no longer important for some reason?

Reply:
The previous implementation stuffed the data and the index into a recarray. The test was necessary to verify that you could transparently build a nested recarray. Now that I'm using tuples of arrays, there's no reason to think recarrays will behave any different from anything else.